### PR TITLE
Add date time and version to config file (fixes #488)

### LINKED ISSUE #488: Add creation date and current kinto version to generated config file
Using comments for example:

```ini
# Created at Thu, 03 Mar 2016 17:02:37 +0100
# Using Kinto version 1.11.2

[server:main]
use = egg:waitress#main
host = 0.0.0.0
port = 8888

```

### PR DESCRIPTION
**Pull request by ayusharma**
Refs  #488

### CODE REVIEW

File: kinto/config/__init__.py [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import os
 import codecs
-
```
Comment (almet) [inline kinto/config/__init__.py]: The spaces are actually legitimate here: in python, you:

- first import standard library modules (hint: time is one)
- then import 3rd party library modules
- then import the modules from you project (here kinto).

Between each block you need an empty line.

**Comment by almet** [comment]
Travis CI is showing the tests that fail. You should see this fail also on your machine (when running `make test`)

File: kinto/tests/test_config.py [hunk 2]
```diff
@@ -68,7 +74,9 @@ def test_init_postgresql_values(self, mocked_render_template):
             'permission_backend': 'cliquet.permission.postgresql',
             'storage_url': postgresql_url,
             'cache_url':  postgresql_url,
-            'permission_url': postgresql_url
+            'permission_url': postgresql_url,
+            'kinto_version': __version__,
+            'config_file_timestamp': strftime('%a, %d %b %Y %H:%M:%S %z')
```
Comment (leplatrem) [inline kinto/tests/test_config.py]: nitpick: for the tests you can use a hard-coded version if you don't test its output

File: kinto/tests/test_configuration/test.ini [hunk 3]
```diff
@@ -1,3 +1,7 @@
+# Created at config_file_timestamp
```
Comment (leplatrem) [inline kinto/tests/test_configuration/test.ini]: Why do you have this `config_file_timestamp` string here ? I thought the file content was compared somewhere else, no ?

**Comment by Natim** [comment]
Thank you @ayusharma :)

File: kinto/tests/test_configuration/test.ini (hunk 3 above)
Comment (Natim) [inline kinto/tests/test_configuration/test.ini]: I tried to change that and tests were failing.

File: kinto/tests/test_config.py (hunk 2 above)
Comment (Natim) [inline kinto/tests/test_config.py]: This is part of testing that the current date is correctly passed to the template.

**Pull request merged**